reference/5.1/Microsoft.PowerShell.Core/About/about_Splatting.md

```diff
@@ -34,7 +32,7 @@ all parameters of a command.
 
 ## SYNTAX
 
-```
+```powershell
```

Comment:
This is a syntax block, not powershell. So remove the language tag.

Reply:
Fixed. Thank you